docs/application/dotnet/samples/general/overview.md

```diff
@@ -1,15 +1,75 @@
 # General
 
+(M) : for mobile profile, (W) : for wearable profile, (T) : for TV profile
 <table>
 	<tbody>
+		<tr>
+			<td><img alt="" height="180" src="media/wplayingwithhwInputs.png" width="180"/></td>
+			<td>
+			<p><strong>(W) PlayingWithHWInputs</strong> [In progress]</p>
+			<p>This sample application demonstrates how to handle rotary events from bezel of a wearable device. In addition, it shows the way how to detect when the back button is pressed.</p>
+			<p>This application uses Tizen.Wearable.CircularUI API.<br>
+			<ul>
+				<li><a href="https://samsung.github.io/Tizen.CircularUI/api/Tizen.Wearable.CircularUI.Forms.IRotaryEventReceiver.html" target="_blank">Interface IRotaryEventReceiver</a></li>
+			</ul></p>
+			<p>Also, you can get details about managing rotary event for Tizen native and web applications.</p>
+			<table>
+				<tbody>
+				<tr>
+					<td>App Type</td>
+					<td>Guide</td>
+				</tr>
+				<tr>
+					<td>Native</td>
+					<td><a href="https://developer.tizen.org/development/guides/native-application/user-interface/efl/hardware-input-handling/managing-rotary-events" target="_blank">Managing rotary events</a></td>
```

Comment:
Please change this link to "../../../native/guides/ui/efl/rotary-events.md".

Reply:
done